WORKSPACE

```diff
@@ -57,6 +57,39 @@ load(
 
 container_repositories()
 
+container_pull(
+    name = "distroless",
+    digest = "sha256:de63da39d0477a9994276cb1de6cec710d9e293ca667ef01ef189b6c87b554e9",
+    registry = "gcr.io",
+    repository = "distroless/static",
+    # tag = "latest",
+)
+
 load("@bazel_gazelle//:deps.bzl", "gazelle_dependencies")
 
 gazelle_dependencies()
+
+git_repository(
+    name = "bazel_skylib",
+    remote = "https://github.com/bazelbuild/bazel-skylib.git",
+    tag = "0.6.0",
+)
+
+git_repository(
+    name = "io_k8s_repo_infra",
+    commit = "9ddd2bf9f38fdcb24709aaa84561de19f0a75cab",
+    remote = "https://github.com/kubernetes/repo-infra.git",
+)
+
+load("//defs:repo_rules.bzl", "fetch_kube_release")
+
+fetch_kube_release(
+    name = "io_k8s_release",
+    archives = {
+        "kubernetes-server-linux-amd64.tar.gz": "025fbdb24d2dce30b83294d075846620437ca393530a606f9527e3cc2ff184b1",
+        "kubernetes-manifests.tar.gz": "429961a7a8b5cd4680a1da06ebd4653b851e1f91a1132b98c9a3f6d01ee9791c",
+        # we do not currently make modifications to these relase tars
```

Comment:
relase -> release.
Also its a little vague in the comment which release tars we do not modify.
I assume you mean the following release tars